ionoscloud/resource_container_registry_token_test.go

```diff
@@ -0,0 +1,232 @@
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckContainerRegistryTokenConfigBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckContainerRegistryTokenExists(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, &containerRegistryToken),
+					// ToDo: fix diff between expiry_date post and get
+					//resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "expiry_date", "2023-01-13T16:27:42Z"),
+					resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.actions.0", "push"),
+					resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.name", "Scope1"),
+					resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.type", "repository"),
+					resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "status", "enabled"),
+					resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "name", ContainerRegistryTokenTestResource),
+				),
+			},
+			{
+				Config: testAccDataSourceContainerRegistryTokenMatchId,
+				Check: resource.ComposeTestCheckFunc(
+					// ToDo: fix diff between expiry_date post and get
+					//resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "expiry_date", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "expiry_date"),
+					resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "scopes.0.actions.0", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.actions.0"),
+					resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "scopes.0.name", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.name"),
+					resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "scopes.0.type", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "scopes.0.type"),
+					resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "status", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "status"),
+					resource.TestCheckResourceAttrPair(DataSource+"."+ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestDataSourceById, "name", ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "name"),
+				),
+			},
+			{
+				Config: testAccDataSourceContainerRegistryTokenMatchName,
+				Check: resource.ComposeTestCheckFunc(
+					// ToDo: fix diff between expiry_date post and get
```

Comment:
remove comment after deciding on a solution regarding the difference between the format sent at create and the one that is received at get